go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/hashicorp/logutils v1.0.0
 	github.com/hashicorp/terraform-exec v0.18.1
 	github.com/hashicorp/terraform-json v0.17.1
-	github.com/hashicorp/terraform-plugin-go v0.18.0
+	github.com/hashicorp/terraform-plugin-go v0.18.1-0.20230824194237-31d190886564
```

Comment:
This will be updated to terraform-plugin-go v0.19.0 pending https://github.com/hashicorp/terraform-plugin-go/pull/311 review, merge, and release.